.travis.yml

```diff
@@ -12,10 +12,12 @@ before_install:
 - pip install -U pip
 - pip install -U setuptools
 - pip install -U wheel
+- pip install -U black
 install:
 - pip install tox-travis .[devel]
 script:
 - tox
+- black --check kojak/
```

Comment:
Please add this to tox to centralize test in an unique file.
By using tox file we can also use this test locally and travis will also call it